hcert_spec.md

```diff
@@ -274,18 +270,19 @@ In a later version, the service may also:
 
 The tasks that are marked _DCCG_ or _CIRCABC_ are expected to be handled by DCCG automation, CIRCABC or similar systems under control and responsibility of the Secretariat.
 
-The format for the lists used for the interchange between the Member States and the Secretariat is waiting for the completion of the T-Systems/SAP proposal -- and should be optimised for clarity and interoperability. The ICAO Master List structure as defined in Doc 9303 part 12 may be considered.
+The format for the lists used for the interchange between the Member States and the Secretariat is available in DGC Gateway's github repository [here](https://github.com/eu-digital-green-certificates/dgc-gateway). The specification is distributed as a build artifact, the latest of which (at time of writing) can be found [here](https://github.com/eu-digital-green-certificates/dgc-gateway/releases/download/1.3.21/openapi.json)
```

Comment:
I prefer avoiding the work `here` for making links. I always prefer to make the links parts of the sentence. 
```markdown
available in [DGC Gateway's github repository](https://github.com/eu-digital-green-certificates/dgc-gateway)
```